src/main/scala/viper/silver/plugin/standard/termination/transformation/FunctionCheck.scala

```diff
@@ -125,7 +126,65 @@ trait FunctionCheck extends ProgramManager with DecreasesCheck with ExpTransform
         }
       } else Nil
     }
-    proofMethods
+    if (respectFuncPermAmounts)
+      proofMethods
+    else
+      proofMethods.map(removeConcretePermissionAmounts)
+  }
+
+  /**
+    * Given a method, removed all concrete permission amounts and replaces them with wildcard if they are positive,
+    * otherwise with none.
```

Comment:
```suggestion
    * Given a method, removes all concrete permission amounts and replaces them with wildcard if they are positive,
    * otherwise with none.
```